Add handling for bad login/no subscription

glennguy opened the pull request:
Incorrect user login and users with no current subscription are handled nicely now. Also set default live video quality to highest.

glennguy commented:
Also added fix for #158

andybotting commented:
I've reworked some of this and added it all in commit cb188d0d87653272c02d4f22edb7a295617b5531

Thanks

glennguy commented:
Awesome, thanks!
On 13 Jun 2016 16:12, "Andy Botting" <notifications@github.com> wrote:

> I've reworked some of this and added it all in commit cb188d0
> <https://github.com/andybotting/xbmc-addon-afl-video/commit/cb188d0d87653272c02d4f22edb7a295617b5531>
>
> Thanks
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/andybotting/xbmc-addon-afl-video/pull/156#issuecomment-225496696>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALW4yNdJrC-AdYBFj5Nw8CgZUyYHiCGnks5qLPTLgaJpZM4Izxbr>
> .
>
